runtime/src/accounts_db.rs

```diff
@@ -651,6 +793,7 @@ impl AccountsDB {
                             &mut reclaims,
                             max_clean_root,
                             &self.account_indexes,
+                            |account_info| account_info.store_id != CACHE_VIRTUAL_STORAGE_ID,
```

Comment:
normally, we now do force_flush => clean_accounts. so there should be no cache account_info for any cached rooted account_infos at the time clean_accounts is running, right?

So, this is needed for older unrooted cached slots?

Reply:
Yeah exactly!